### PR TITLE
Updated the Getting Started guide

### PR DESCRIPTION
**Pull request by Jimvin**

### CODE REVIEW

File: modules/ROOT/pages/getting_started.adoc [hunk 1]
```diff
@@ -0,0 +1,407 @@
...
+    kafka-simple-default-broker-node2       1/1     Running      0          6m32s
+    nifi-simple-default-node-node2          1/1     Running      0          6m32s
+    kafka-simple-default-broker-node1       1/1     Running      0          6m32s
+    nifi-simple-default-node-node1          1/1     Running      0          6m32s
+    zookeeper-simple-default-server-node1   1/1     Running      0          6m32s
+
+Since this is the first time that each of these services has been deployed to these nodes then Stackable Agent needs to download the software from the Stackable repository. It may take a few minutes to complete the download and deploy the services.
+
+== Testing your cluster
+If all has gone well then you will have successfully deployed a Stackable cluster.
```
Comment (soenkeliebau) [inline modules/ROOT/pages/getting_started.adoc:368]:
```suggestion
If all has gone well then you will have successfully deployed a Stackable cluster and used it to start three services that should now be ready for you.
```

**Review by soenkeliebau — CHANGES_REQUESTED** [review]
Some minor things, but overall great work!

**Review by Jimvin — COMMENTED** [review]
I've incorporated the suggestions in the latest updates.

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -85,319 +85,8 @@ Rust is a Programming language, which is focused on safety. It provides memory s
...
-You can do this by running `kubectl certificate approve <agent-hostname>-tls` on the orchestrator server after starting the agent.
-
-If your Kubernetes does not offer certificate signing services, you can alternatively also manually generate a keypair and sign a certificate with a proper authority.
-These files will then need to be configured in the agent config file.
-
-=== Health Check
-After performing the steps outlined above you should have an empty Stackable Platform up and running.
-When you run `kubectl get nodes` on the orchestrator, you should see the objects that all your agents added to indicate their readiness.
+Stackable provides a tutorial link:getting_started.adoc[quickstart guide], which outlines how to deploy a Stackable cluster and the supported infrastructure. There is also a quickstart shell script in the https://github.com/stackabletech/stackable-utils[stackable-utils] GitHub repository that will quickly bootstrap an example cluster.
+d and enabled.
```
Comment (soenkeliebau) [inline modules/ROOT/pages/index.adoc:89]: Was this intentional?

**Review by soenkeliebau — CHANGES_REQUESTED** [review]
Just two really minor things

**Review by Jimvin — COMMENTED** [review]
All the outstanding recommendations have been actioned, should be good to go.

File: modules/ROOT/pages/index.adoc (hunk 2 above)
Comment (Jimvin) [reply modules/ROOT/pages/index.adoc]: Removed the trailing sentence fragment.

**Review by soenkeliebau — APPROVED** [review]

**Pull request merged**